organizations/user.go

```diff
@@ -160,8 +160,8 @@ func CreateMember(w http.ResponseWriter, r *http.Request) {
 		Role:     "member",
 		Presence: "true",
 		JoinedAt: time.Now(),
-		Settings: utils.M{},
-		Deleted:  false,
+		// Settings: utils.M{},
```

Comment:
if you comment out the settings, then how do you assign default setting to a new member?